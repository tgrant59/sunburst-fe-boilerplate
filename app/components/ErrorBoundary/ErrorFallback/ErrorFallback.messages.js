export default {
    errorFallback: 'Uh oh! Something went wrong here',
}
