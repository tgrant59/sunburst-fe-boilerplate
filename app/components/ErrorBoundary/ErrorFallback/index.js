import React from 'react'

import messages from './ErrorFallback.messages'

const ErrorFallback = () => <p>{messages.errorFallback}</p>

export default ErrorFallback
