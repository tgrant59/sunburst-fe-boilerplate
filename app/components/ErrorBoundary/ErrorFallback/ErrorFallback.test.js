import React from 'react'
import { mount } from 'enzyme'

import ErrorFallback from '.'

describe('<ErrorFallback />', () => {
    it('should match snapshot', () => {
        const renderedComponent = mount(<ErrorFallback />)
        expect(renderedComponent).toMatchSnapshot()
    })
})
