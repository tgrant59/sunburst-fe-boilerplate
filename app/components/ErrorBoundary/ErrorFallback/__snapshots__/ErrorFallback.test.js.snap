// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorFallback /> should match snapshot 1`] = `
<ErrorFallback>
  <p>
    Uh oh! Something went wrong here
  </p>
</ErrorFallback>
`;
